src/com/generic/main/Shipment.java

```diff
@@ -0,0 +1,48 @@
+package com.generic.main;
+
+public class Shipment {
+	private String shipmentID;
+	private String freight;
+	private float weight;
+	private int receiptDate; //Need to figure out the date format
+	
+	public Shipment(String id, String f, float w, int d) {
+		shipmentID = id;
```

Comment:
naming conventions of arguments should be readable as codes should self-document itself.